Common/tests/Pollster/TaskHandlerTest.cs

```diff
@@ -964,6 +964,43 @@ public async Task AcquireTaskFromCancelledSessionShouldFail()
                                        .TaskId);
   }
 
+  [Test]
+  public async Task AcquireTaskFromPausedSessionShouldFail()
```

Comment:
I thought you had a framework to test the acquisition. Can't you use it for this test?

Reply:
It does not manage session data so I did like the other test on the subject